source/material_model/visco_plastic.cc

```diff
@@ -267,7 +267,15 @@ namespace aspect
               }
             }
 
-          // Step 1d: multiply the viscosity by a constant (default value is 1)
+          // Step 1d: compute viscosity from Peierls creep law and harmonically average with current viscosities
+          double viscosity_peierls = numbers::signaling_nan<double>();
```

Comment:
You do not need this outside the `if`. Make line 274 into `const double viscosity_peierls = ...`